tests/test_filepath.py

```diff
@@ -93,32 +92,30 @@ def test_duplicate_error_s3():
 
 def test_filepath_class(table=Filepath(), store="repo"):
     stage_path = dj.config['stores'][store]['stage']
-    filename = 'attachment.dat'
-
     # create a mock file
     relative_path = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relative_path), exist_ok=True)
-    managed_file = os.path.join(stage_path, relative_path, filename)
+    managed_file = Path(stage_path, relative_path, 'attachment.dat')
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `C:\\Users\\rguzman\\AppData\\Local\\Temp\\tmpb3gvp0ht\\one\two\threeattachment.dat`.